apps/fhir/bluebutton/v2/urls.py

```diff
@@ -0,0 +1,39 @@
+from django.conf.urls import url
+from django.contrib import admin
+
+from apps.fhir.bluebutton.views.read import ReadViewCoverage, ReadViewExplanationOfBenefit, ReadViewPatient
+from apps.fhir.bluebutton.views.search import SearchViewCoverage, SearchViewExplanationOfBenefit, SearchViewPatient
+
+admin.autodiscover()
+
+urlpatterns = [
+    # Patient ReadView
+    url(r'Patient/(?P<resource_id>[^/]+)',
+        ReadViewPatient.as_view(),
+        name='bb_oauth_fhir_patient_read_or_update_or_delete_v2'),
+
+    # Patient SearchView
+    url(r'Patient[/]?',
+        SearchViewPatient.as_view(),
+        name='bb_oauth_fhir_patient_search_v2'),
```

Comment:
@JFU-GIT What are your thoughts about moving the version up to the Django (URL) routing layer to be more flexible?

Here is an example using the Patient FHIR resource's `SearchViewPatient` class:

```suggestion
    # Patient SearchView
    url(r'Patient[/]?',
        SearchViewPatient.as_view(version=2),
        name='bb_oauth_fhir_patient_search_v2'),
```